### ISSUE TITLE
GDS project.cypher progressbar only takes node loading into account?

### ISSUE THREAD

**Issue by daveaitel** [neo4j/graph-data-science-client#350]
graphdatascience   1.6

I'm running a fairly large query:
```

    print("Projecting graph %s" % graphname)
    #this next line is fairly slow - "unlicensed GDS" can only support a readConcurrency of 4 or less. We have to have the
    #relationship validation turned off because some f2's are not included in the repo! But if we do a join of those two
    #sets, it will take literally forever. 
    #I've looked into it though - and if you are a GDS licensed user you can set readConcurrency higher,
    #but that's basically the only way to improve this (sorry)
    G, res = gds.graph.project.cypher(graphname, "MATCH (repo:Repo {db_key: $db_key})<-[:INCLUDED]-(f:File) RETURN id(f) as id",
                                      """MATCH (repo:Repo {db_key: $db_key})<-[:INCLUDED]-(f:File)-[r:CONNECTED_TO]->(f2:File)
                                         RETURN id(f) as source, id(f2) as target, type(r) as type, r.weight as weight""",
                                      parameters = params, validateRelationships=False)
    print("Projected graph %s" % graphname)
```

What I see obviously is this:
![image](https://user-images.githubusercontent.com/20305506/234360645-3b4fa2fe-d2f5-44a3-b897-1eb5bed13b1c.png)

If I was able to attach an animation you'd see the bar go to 100% and then sit there for as long as it takes the second part of the query to run (which can be a very long time). 

Perhaps I'm wrong about this but my feeling is the progressbar is tied to the node loading and not the relationship loading!

Thanks!

**Comment by FlorentinD** [comment]
Hello @daveaitel ,
sorry for taking so long to reply.
Which server version are you using? We recently fixed the progress bar in `2.3.2` (https://github.com/neo4j/graph-data-science/releases/tag/2.3.2).
Assuming you are using an earlier version, you are right, the relationship query was not correctly considered.

**Comment by Mats-SX** [comment]
Closing on account of inactivity.
